packages/cozy-client/src/models/bitwarden.js

```diff
@@ -0,0 +1,26 @@
+import { Q } from '../queries/dsl'
+import CozyClient from '../CozyClient'
+
+/**
+ * Checks the value of the extension_installed attribute
+ *
+ * @param {CozyClient} client - The CozyClient instance
+ * @returns {Promise<boolean>} - Returns the value of the extension_installed attribute
+ */
+export const hasExtensionInstalledAttribute = async client => {
```

Comment:
why the `attribute` stuff?

Reply:
We have added `attribute` to avoid confusion between hasPassword and hasPasswordDefined and we have followed the same convention for hasExtensionInstalled